### PR TITLE
Add blog post about why adopting Tuist is a good idea to make scaling up projects easy

### PR DESCRIPTION
**Pull request by pepibumur**
As the title says, I'm adding a blog post that I've been aiming to write for a long time, and I finally wrote it up.

### CODE REVIEW

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+If you are considering the adoption of Tuist,
+you might wonder _why_ a project like yours should make such a move.
+The idea of adding a new tool to your toolbox might sound intimidating ―
+it certainly is.
+But believe me,
+this is one that you won't regret about.
+Despite having done our best to convey the idea behind Tuist,
+and why it's an important piece when scaling up projects,
+I feel we lacked a good summary
```
Comment (natanrolnik) [inline website/markdown/posts/2020-04-28-why-tuist/post.mdx:27]: Missing a period here

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+That makes it possible to create a new feature framework that is consistent with the rest.
+For instance,
+I can create a new Search feature framework by creating a `Search` directory,
+and placing the following `Project.swift` file in it:
+
+```swift
+import ProjectDescription
+import ProjectDescriptionHelpers
+
+let project = Project.featureFramework(name: "Seach")
```
Comment (natanrolnik) [inline website/markdown/posts/2020-04-28-why-tuist/post.mdx:70]: I think you have a typo here. Shouldn’t it be Search?

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+Here are some of the features that we are planning,
+and that you'll be able to opt-in easily if you are already using Tuist:
+
+- **Caching:** This is one of the features we are the most excited about building.
+  Unlike build systems like [Buck](https://github.com/facebook/buck) or [Bazel](https://bazel.build/),
+  which are mostly adopted by large companies that can invest in build systems and tooling,
+  we are exploring adding caching at the module level.
+  We'll generate projects where only the targets you plan to work on will be generated ―
+  the rest will be pre-compiled frameworks and libraries that we'll pull from a remote cache.
+  And the beauty of this feature is that you'll be able to opt-in easily if you are already using Tuist.
```
Comment (natanrolnik) [inline website/markdown/posts/2020-04-28-why-tuist/post.mdx:154]: It’s important to write this, but sounds a bit repetitive (this is mentioned a few lines above). Maybe change the words?

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+There are also build phases, targets, or schemes, for which Xcode doesn't provide any way to reuse them.
+Other teams resort to scripts that run some checks on the projects,
+but that results in a poor experience for developers because those checks are not built into their workflows.
+
+Tuist solves this by providing [project description helpers](/docs/usage/helpers/).
+All you need to do is to define what types of projects are supported,
+and codify them into functions that return templated projects:
+
+```swift
+imporrt ProjectDescription
```
Comment (sgrgrsn) [inline website/markdown/posts/2020-04-28-why-tuist/post.mdx:51]: There's a r too many in the import

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx (hunk 2 above)
Comment (pepibumur) [reply website/markdown/posts/2020-04-28-why-tuist/post.mdx:70]: Right. Good catch 👁️

File: website/markdown/posts/2020-04-28-why-tuist/post.mdx (hunk 4 above)
Comment (pepibumur) [reply website/markdown/posts/2020-04-28-why-tuist/post.mdx:51]: `imporrt` is the new way of importing things :trollface:

**Comment by sgrgrsn** [comment]
Great blog post 👍 
When I'm talking to people about Tuist the first question that usually pops up is something like:  _"What happens if Tuist is no longer maintained - will you then still be able to build your project?"_

I think that would be a great topic to cover in this blog post as well.

**Comment by pepibumur** [comment]
> When I'm talking to people about Tuist the first question that usually pops up is something like: "What happens if Tuist is no longer maintained - will you then still be able to build your project?"

I love that! I thought about adding a section around that topic to this blog post but I think it deserves its own.

**Pull request merged**